WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:20:32] "GET /healthz HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:21:13] "GET /healthz HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:21:13] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:21:50] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:22:12] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:22:21] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:22:24] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:23:11] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:25:06] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:25:11] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 61, in index
    posts = connection.execute('SELECT * FROM posts').fetchall()
sqlite3.OperationalError: no such table: posts
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:25:18] "[35m[1mGET / HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:25:27] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 61, in index
    posts = connection.execute('SELECT * FROM posts').fetchall()
sqlite3.OperationalError: no such table: posts
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:25:52] "[35m[1mGET / HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 11:26:00] "[35m[1mGET /healthz HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:21] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:22] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:24] "GET /create HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:24] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:32] "[32mPOST /create HTTP/1.1[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:32] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:32] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:43] "GET /healthz HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:47:53] "[33mGET /metrics HTTP/1.1[0m" 404 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 16:48:53] "GET /metrics HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:12:25] "GET /healthz HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:49:07] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:49:07] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:49:38] "GET /healthz HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:49:43] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:49:49] "GET /metrics HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 47, in metrics
    "post_count": post_count
NameError: name 'post_count' is not defined
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:56:36] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 47, in metrics
    "post_count": count_posts
NameError: name 'post_count' is not defined
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:57:13] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 17:57:37] "GET /metrics HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 47, in metrics
    "db_connection_count": app.config['connection_count'],
KeyError: 'connection_count'
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:02:08] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 47, in metrics
    "db_connection_count": app.config[connection],
KeyError: <sqlite3.Connection object at 0x7fa6bb0903b0>
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:04:59] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 47, in metrics
    "db_connection_count": app.config[connection],
KeyError: <sqlite3.Connection object at 0x7fa6bb0901f0>
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:05:02] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:15:03] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:15:03] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 44, in metrics
    db_connection_count = connection.execute('SELECT SUM(article_view) FROM posts').fetchall()
sqlite3.OperationalError: no such column: article_view
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:15:09] "[35m[1mGET /metrics HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.86.146:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:28] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:28] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:33] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:41] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:44] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:46] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:47] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:49] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:50] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:52] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:53] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:55] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [21/Mar/2023 18:28:56] "GET /metrics HTTP/1.1" 200 -
